sparql/disease-gene-relationships.sparql

```diff
@@ -24,6 +24,7 @@ WHERE {
 
   FILTER(
     ?PredUri IN (
+      <http://purl.obolibrary.org/obo/RO_0003302>,
```

Comment:
For completeness, good to add in case any analysis comparisons of the data ever look at the other gene association properties other than the disease-defining property.